.github/workflows/compliance.yml

```
@@ -0,0 +1,16 @@
+name: compliance
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  # Reference: ENGSRV-059
+  copywrite:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      - uses: hashicorp/setup-copywrite@867a1a2a064a0626db322392806428f7dc59cb3e # v1.1.2
+      - run: copywrite headers --plan
```

Let's also enable `LICENSE` file checking while we're here 😄 

```suggestion
      - run: copywrite headers --plan
      - run: copywrite license
```

It will return a non-zero exit status if there's an issue with the file:

```
$ copywrite license
Licensing under the following terms: MPL-2.0
Using year of initial copyright: 2021
Using copyright holder: HashiCorp, Inc.

Validated file: LICENSE
Validating presence of license header
2023-06-02T09:50:08.804-0400 [ERROR] cli: license file has a copyright statement, but it is malformed; Expected to find: "Copyright (c) 2021 HashiCorp, Inc." Please resolve this manually
Error: license file has a copyright statement, but it is malformed; Expected to find: "Copyright (c) 2021 HashiCorp, Inc." Please resolve this manually
$ echo $?
1
```

---

Good call, added! ⚡